config.default.yaml

```diff
@@ -151,6 +151,19 @@ load_options:
   prediction_year: 2030 # Load scenarios available with different prediction year (GDP, population)
   scale: 1 # scales all load time-series, i.e. 2 = doubles load
 
+co2_budget:
```

Comment:
Could you please document those parameters adding a config table to [the existing ones](https://github.com/pypsa-meets-earth/pypsa-earth/tree/main/doc/configtables)?